src/app/pages/settings/settings.page.ts

```diff
@@ -1,43 +0,0 @@
-import { Component, OnInit } from '@angular/core';
-import { Storage } from '@ionic/storage';
-
-import { Events } from '@ionic/angular';
-
-@Component({
-  selector: 'app-settings',
-  templateUrl: './settings.page.html',
-  styleUrls: ['./settings.page.scss'],
-})
-export class SettingsPage implements OnInit {
-    pushEnabled: any;
-
-  constructor(private storage: Storage,
-              public events: Events) {
-
-  }
-
-  async ngOnInit() {
-    await this.storage.ready();
-    const pushEnabledInStorage = await this.storage.get('pushEnabled');
-
-    // Enable Push if there is no configuration in storage from user yet
-    if (pushEnabledInStorage === null || pushEnabledInStorage) {
```

Comment:
Very small thing (probably comms issue) 

There is no way now to switch push on/off and pushService still have code to detect if push is disabled. I think by default push will be enabled so there is no issue with switch, but we should cleanup ` this.events.subscribe('settings:changed')`

Reply:
Ah right, that makes sense. Do we still want to be able to switch push on and off?

Reply:
Not specially. But we should remove the code that will read this property as it will be really confusing for people to see that there is entire logic for reaching from storage but we never write that anywhere.